Servers/UI/OJS.Servers.Ui/ClientApp/src/layout/content/PageContent.module.scss

```diff
@@ -2,6 +2,7 @@
   display: flex;
   flex-direction: column;
   min-height: 100vh;
+  padding-bottom: 75px;
   // padding-bottom: 75px;

```

Comment:
remove comment here and check if there is suitable variable for the 75px to be used, if not leave it like that, and also please check the vh if it doesnt break in responsive behaviour

Reply:
I've fixed the spacings values

![image](https://github.com/SoftUni-Internal/OpenJudgeSystem/assets/19411274/5401a5d8-9873-472b-9adb-e071ddc216c7)
![image](https://github.com/SoftUni-Internal/OpenJudgeSystem/assets/19411274/17faee7d-0a08-4475-a955-75dde2c77472)


There seems to be a general issue with height/width on smaller screens so i suggest we fix this in a different issue